src/common/verify/isValidAccessGrant.test.ts

```diff
@@ -137,6 +152,31 @@ describe("isValidAccessGrant", () => {
     );
   });
 
+  it("sends the given plain JSON vc to the verify endpoint", async () => {
+    jest.mocked(isVerifiableCredential).mockReturnValueOnce(true);
+    jest.mocked(getVerifiableCredentialApiConfiguration).mockResolvedValueOnce({
+      verifierService: "https://some.vc.verifier",
+      specCompliant: {},
+      legacy: {},
+    });
+    const mockedFetch = jest.fn<typeof fetch>().mockResolvedValueOnce(
+      new Response(JSON.stringify(MOCK_VERIFY_RESPONSE), {
+        status: 200,
+      }),
+    );
+
+    await isValidAccessGrant(MOCK_ACCESS_GRANT, {
```

Comment:
```suggestion
    await isValidAccessGrant(MOCK_ACCESS_GRANT_BASE, {
```

Reply:
I tried with that, but since `isValidAccessGrant` input allows types are either `string`, `VerifiableCredential`, `DatasetWithId` or `URL`, with `MOCK_ACCESS_GRANT_BASE` being just an object, it raises a TS error.

Reply:
Oh I see, then stringify and parse is fine, or you can do `as any`, which is equivalent. ESlint will complain about it, you can add an ignore mentioning we are explicitly testing for a behavior that is not compliant with our API type definitions.